resources/ui/src/main/resources/AdminGuide/Integration.xml

```diff
@@ -49,72 +62,73 @@ The default skin of PhenoTips is not well suited for embedding into another appl
 * Log in with an administrative account
 * Click on the //Administration// link displayed in the top left corner
 * Select the //Look &amp; Feel / Presentation// entry from the left menu
-* Type ##PhenoTips.EmbeddableSkin## in the //Skin// field, click //Save//
-
-{{warning}}Before 1.0-milestone-13, you must do the same for the //data// space: once you're in the administrative area, from the drop down menu in the upper right side, currently displaying ##Wiki administration##, select ##data##, and change the //Skin// option here as well.{{/warning}}
+* Type **PhenoTips.EmbeddableSkin** in the //Skin// field, click //Save//
 
 You should now see only the essential elements of the page. Unfortunately, this doesn't include links back to the homepage, or to the administration. Fortunately, you can configure the skin only for a specific user, too, though the UI for doing that is not easily discoverable:
 
-* Open ##**(% style="color: #F93" %)http(%%)**:/~/**(% style="color: #900" %)localhost(%%)**(% style="color: #009" %)**:8080**(%%)/**(% style="color: #990" %)phenotips/(%%)**bin/edit/XWiki/**(% style="color: #090" %)Admin(%%)**?editor=object&amp;classname=XWiki.XWikiUsers&amp;object=0&amp;property=skin## - using the right **(% style="color: #F93" %)protocol(%%)**, **(% style="color: #900" %)host name(%%)**, **(% style="color: #009" %)optional port(%%)**, **(% style="color: #990" %)optional application name(%%)**, and the right **(% style="color: #090" %)username(%%)**
-* Type ##PhenoTips.Skin## in the //skin// field, click one of the //Save// buttons
+* Open **(% style="color: #F93" %)http(%%)**:/~/**(% style="color: #900" %)localhost(%%)**(% style="color: #009" %)**:8080**(%%)/**(% style="color: #990" %)phenotips/(%%)**bin/edit/XWiki/**(% style="color: #090" %)Admin(%%)**?editor=object&amp;classname=XWiki.XWikiUsers&amp;object=0&amp;property=skin - using the right **(% style="color: #F93" %)protocol(%%)**, **(% style="color: #900" %)host name(%%)**, **(% style="color: #009" %)optional port(%%)**, **(% style="color: #990" %)optional application name(%%)**, and the right **(% style="color: #090" %)username(%%)**
+* Type **PhenoTips.Skin** in the //skin// field, click one of the //Save// buttons
 
 == Displaying a patient record ==
 
-Display ##http[s]://&lt;hostname&gt;[:&lt;port&gt;]/&lt;appname&gt;/bin/PhenoTips/OpenPatientRecord?[e]id=&lt;identifier&gt;[&amp;mode=&lt;mode&gt;][&amp;create=true]## in an iframe.
+Display **http[s]:/~/&lt;hostname&gt;[:&lt;port&gt;]/&lt;appname&gt;/bin/PhenoTips/OpenPatientRecord?[e]id=&lt;identifier&gt;[&amp;mode=&lt;mode&gt;][&amp;create=true]** in an iframe.
 
 Where:
-; ##hostname##
+; **hostname**
 : the actual hostname where PhenoTips is reachable
-; ##port##
+; **port**
 : the port, if needed
-; ##appname##
+; **appname**
```

Comment:
So, I originally had an outer velocity layer to display different things to authenticated users, which required changing all of these because `##` caused them to be ignored by velocity before being parsed by the wiki-markup. I left those changes in case we decide to use velocity again later.